R/FilteredDatasetDefault.R

```diff
@@ -76,7 +83,7 @@ DefaultFilteredDataset <- R6::R6Class( # nolint
     #' @description
     #' Get `data.frame` to use in the filter-panel overview.
     #' Output of this function indicates unsupported dataset class.
-    #' @return `list` containing character `#filtered/#not_filtered`
+    #' @return A `data.frame`.
```

Comment:
It would be nice to give explanation of what's inside. The structure isn't complex.
(this applies for all the `get_filter_overview` method)

Reply:
Is the description not sufficient? I tried to keep `@return` to the type.

Reply:
For _simple_ (atomic) types of data (and simple methods) I guess that only the type is sufficient - good examples are: `get_label`, `get_call` etc.
`data.frame` is usually a collection of information - it would be nice to explain it a little what's inside as long as the structure is not too complex not to do so. I already did a similar thing explaining the structure of teal_slices object somewhere above

Reply:
Fair enough, will elaborate.

Reply:
58059e5217ce08cc6ee2e69737e06b7d6f4a4673